microsoft-365/compliance/customer-key-manage.md

```diff
@@ -143,6 +143,10 @@ To create a DEP, you need to remotely connect to SharePoint Online by using Wind
    ```powershell
    Register-SPODataEncryptionPolicy -PrimaryKeyVaultName 'stageRG3vault' -PrimaryKeyName 'SPKey3' -PrimaryKeyVersion 'f635a23bd4a44b9996ff6aadd88d42ba' -SecondaryKeyVaultName 'stageRG5vault' -SecondaryKeyName 'SPKey5' -SecondaryKeyVersion '2b3e8f1d754f438dacdec1f0945f251a’
    ```
+For a managed HSM run the Register-SPODataEncryptionPolicy cmdlet as follows:
```

Comment:
```suggestion
For a Managed HSM Key Vault, run the Register-SPODataEncryptionPolicy cmdlet as follows:
```

Reply:
I don't think Managed needs to be capped.